CommandLine/CommandLine.swift

```diff
@@ -79,7 +79,16 @@ public class CommandLine {
 
     return usedFlags
   }
-
+    
+  /**
+   * If true, when the stopper is encountered '--', we not only stop parsing options in general
+   * we also stop parsing multi-value options. This allows a user to indicate to an unbounded option
+   * to stop if it happens to be the last option in the option list, just before the non-option
+   * unparsed arguments.
+   *
+   */
+  public var stopperStopsMultiValueOptionParsing = false
```

Comment:
Yeah, I don't think that's necessary in a command line tool. I think the way that would usually be supported is to put them in quotes or something. I think users generally expect `--` to mean "I'm all done with options".